From c0512eb26a021ef6256cfaa541ca7cde17c67d00 Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Sat, 28 Mar 2020 08:08:28 +0530
Subject: [PATCH 511/767] coresight: Correct OcteonTx2 QUIRK_BUFFSIZE_8BX
 applicability

commit 60df79f9673cac86e6f3d4ce01be8259ae057043 from
git@git.assembla.com:cavium/WindRiver.linux.git

CSETR_QUIRK_BUFFSIZE_8BX is applicable only to OcteonTx2
96xx and 95xx.

Change-Id: If179ab232942592d671e8b3ea0e439680b102a07
Signed-off-by: Linu Cherian <lcherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/26059
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/hwtracing/coresight/coresight-quirks.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwtracing/coresight/coresight-quirks.c b/drivers/hwtracing/coresight/coresight-quirks.c
index f1d2e10ff2b4..5dd3652facc7 100644
--- a/drivers/hwtracing/coresight/coresight-quirks.c
+++ b/drivers/hwtracing/coresight/coresight-quirks.c
@@ -16,12 +16,12 @@ u32 coresight_get_etr_quirks(unsigned int id)
 				     MIDR_CPU_VAR_REV(0, 0),
 				     MIDR_CPU_VAR_REV(2, 0)))
 		options |= CSETR_QUIRK_RESET_CTL_REG |
+			CSETR_QUIRK_BUFFSIZE_8BX |
 			CSETR_QUIRK_NO_STOP_FLUSH;
 
 	/* Common across all Chip variants and revisions */
 	if (id == OCTEONTX_CN9XXX_ETR)
 		options |= CSETR_QUIRK_SECURE_BUFF |
-			CSETR_QUIRK_BUFFSIZE_8BX |
 			CSETR_QUIRK_FORCE_64B_DBA_RW;
 
 	return options;
-- 
2.31.1

